test/unit/test-performance.js

```diff
@@ -1080,87 +1195,10 @@ describes.realWin('PeformanceObserver metrics', {amp: true}, env => {
   });
 
   describe('forwards layout jank metric', () => {
-    let fakeWin;
-    let windowEventListeners;
-    let performanceObserver;
-    let viewerVisibilityState;
-
     beforeEach(() => {
-      // Fake window to fake `document.visibilityState`.
-      fakeWin = {
```

Comment:
Ah, it was moved from here. If it's not easy to migrate to `FakeWindow`, just leave a TODO above.

Reply:
I added a TODO here. We want to control the value of `window.document.visibilityState`, which is not immediately possible with our window fake. Let's follow-up on this.